bootstrap/http/src/main/kotlin/com/studentcenter/weave/bootstrap/meeting/dto/PendingMeetingScrollRequest.kt

```diff
@@ -20,13 +20,12 @@ data class PendingMeetingScrollRequest(
     limit = limit,
 ) {
 
-    fun toCommand(): ScrollPendingMeetingUseCase.Command {
-        return ScrollPendingMeetingUseCase.Command(
+    fun toCommand(): ScrollPendingMeeting.Query {
+        return ScrollPendingMeeting.Query(
```

Comment:
p4) 이것도 `toCommend` 아니고 `toQuery`로?
```suggestion
    fun toQuery(): ScrollPendingMeeting.Query {
        return ScrollPendingMeeting.Query(
```

Reply:
매의 눈 ㄷㄷ